src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import cn from 'classnames';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+type Props = {
+  people: Person[] | [],
```

Comment:
It can be an empty array as well
```suggestion
  people: Person[],
```